src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -114,7 +114,7 @@ private function validate($content, $file = null)
         try {
             $this->getParser()->parse($content, Yaml::PARSE_CONSTANT);
         } catch (ParseException $e) {
-            return array('file' => $file, 'valid' => false, 'message' => $e->getMessage());
+            return array('file' => $file, 'line' => $e->getParsedLine(), 'valid' => false, 'message' => $e->getMessage());
```

Comment:
`getParsedLine()` can return `-1`. Should we add special handling for that case?